web/styles/recent_topics.css

```diff
@@ -72,26 +72,33 @@
             opacity: 0.6;
         }
 
+        #recent_filters_group {
+            width: 100%;
+            white-space: nowrap;
+            overflow-x: auto;
+        }
+
         .table_fix_head {
             padding: 0 !important;
-            /* 100px = space occupied by `recent_topics_filter_buttons`( ~49px)
-                     + give user some extra space at the bottom so that last
-                       topic row is clearly visible. */
-            max-height: calc(100vh - 100px);
+            /* 100vh - height of #recent_topics_filter_buttons - 12px padding of #recent_topics_table
+            - height of navbar - height of compose box */
+            max-height: calc(
+                100vh - 87.2px - 12px - var(--header-height) -
+                    var(--compose-box-height)
+            );
```

Comment:
@timabbott thoughts on doing this in `resize.js`?

Reply:
We can get the filter button height their correctly. 

```
const recent_topics_filters_height = $("#recent_topics_filter_buttons").safeOuterHeight(
            true,
        );
```

Reply:
Also, we should use `--navbar-fixed-height`.

Reply:
or, since the only variable factor here is the filter buttons height, we can use a varibale for it `--recent-conversation-filters-height` and update it in resize.js.

Reply:
Probably that last proposal is the best option, though ideally we'd just do it with pure CSS somehow.